scss/_patterns_suru.scss

```diff
@@ -0,0 +1,20 @@
+@import 'settings';
+
+@mixin vf-p-suru {
+  .p-suru {
+    aspect-ratio: 5.1775700934579; // exact aspect ration of the image
```

Comment:
That's a lot of significant numbers!

We could instead use the size of the image or the reduced fraction
![image](https://github.com/canonical/vanilla-framework/assets/11927929/15af3d36-b873-4af3-b93d-30702cfae36c)


so like `aspect-ratio: 554/107;`

Because what we're actually doing here is making the calculation ourselves and then telling the browser that the ratio is `5.1775700934579 / 1`